.travis.yml

```diff
@@ -131,6 +132,21 @@ matrix:
         on:
           branch: master
 
+    - os: windows
+      # language python currently not supported on Windows
+      language: cpp
+      compiler: msvc
+      git:
+        clone: false
+      before_install:
+        - export -f travis_fold travis_nanoseconds travis_time_start travis_time_finish
+        - choco install python3 --version=3.7.5 --params "/InstallDir:C:\\Python"
+        - choco install swig
+        - export PATH="/c/Python:/c/Python/Scripts:/c/Users/travis/AppData/Roaming/Python/Python38/Scripts:$PATH"
```

Comment:
We are installing python 3.7.5, but this path is for python 3.8. Shouldn't be doing anything, so can probably be removed.

Reply:
Does that mean the these 4 lines should be removed? Or just line 143?

Reply:
- export PATH="/c/Python:/c/Python/Scripts:/c/Users/travis/AppData/Roaming/Python/Python38/Scripts:$PATH"

Reply:
The line has  been deleted.